src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+abstract class Figure implements IFigure {
```

Comment:
The class `Figure` should not be abstract if it is meant to set behavior for classes. Consider using an interface to define behavior or a concrete class if it should provide some common implementation details.